Solutions/Marain.Operations.Hosting.AspNetCore/Microsoft/Extensions/DependencyInjection/OperationsRepositoryServiceCollectionExtensions.cs

```diff
@@ -31,7 +32,7 @@ public static class OperationsRepositoryServiceCollectionExtensions
             services.AddRootTenant();
 
             services.AddSingleton(sp => sp.GetRequiredService<IConfiguration>().GetSection("TenancyClient").Get<TenancyClientOptions>());
-            services.AddAzureManagedIdentityBasedTokenSource();
+            services.AddAzureManagedIdentityBasedTokenSource(default(AzureManagedIdentityTokenSourceOptions));
```

Comment:
This evaluates to `null`, so presumably this means anything asking for `AzureManagedIdentityTokenSourceOptions` as a dependency gets `null`. Is that what you meant?

Reply:
The intention is to pass in `null` here (although if you pass in `null` directly, it complains that the call is ambiguous between overloads, hence the use of `default(AzureManagedIdentityTokenSourceOptions)`).

However, we are not registering `AzureManagedIdentityTokenSourceOptions` in the container; it is only being used to configure the instance of `AzureManagedIdentityTokenSource` that gets registered (see [here](https://github.com/corvus-dotnet/Corvus.Identity/blob/master/Solutions/Corvus.Identity.ManagedServiceIdentity.ClientAuthentication/Microsoft/Extensions/DependencyInjection/ServiceIdentityServiceCollectionExtensions.cs)). `AzureManagedIdentityTokenSourceOptions` just serves as a way to pick up `AzureServicesAuthConnectionString` from configuration - if it is null, then `AzureManagedIdentityTokenSource` will fallback to default behaviour for acquiring a token.